docs/How-to-setup-mysql-plugin.md

```diff
@@ -0,0 +1,14 @@
+# How to setup Mysql plugin for OMS agent
+
+How to enable mySQL plugin to collect mySQL server stats and perf data
+* Machine has to have libmysqlclient.so installed (which can be downloaded and installed through mysql-devel.
+					yum install mysql-devel or apt-get mysql-devel for Debian systems)
```

Comment:
Fixed the readme. Most debian systems come with this library. Also installing through MySQL-devel would not be always used as an option in production environment. So I have updated the readme to have just one example for installing it through devel package.